oneanddone/tasks/templates/tasks/detail.html

```diff
@@ -123,12 +123,12 @@
 
       <form method="post" action="{{ url('tasks.abandon', attempt.id) }}">
         {{ csrf() }}
-        <button type="submit" class="button" id="abandon-task">{{ _('Abandon task') }}</button>
+        <button type="submit" title="{{ _('Abandon this task') }}" class="button" id="abandon-task">{{ _('Abandon this task') }}</button>
```

Comment:
@tessie The word `this` was only meant to be added to the tooltip, not to the button text, so this is incorrect. Can you please fix it and issue a new PR?

The same goes for line 131 below.